app/javascript/packs/outdoor_map_builder.js

```diff
@@ -334,12 +334,15 @@ window.routingControl = L.Routing.control({
 })
 .on('waypointschanged', (e)=>{
   console.log("waypointschanged");
-  // this handler is called whenever the waypoints are changed in any way (search bar or clicking in the map)
+  
+  // we only highlight the destination of the current navigation route
   changeHighlightedBuilding(routingControl.getWaypoints()[1].latLng);
 
+  // this handler is called whenever the waypoints are changed in any way (search bar or clicking in the map)
```

Comment:
Why did you move this comment?

Reply:
> Why did you move this comment?

I was there when @23mafi changed it, so I can say something about that: a previous PR moved some stuff around, and some code was misaligned with comments after that.